bwc-test/build.gradle

```diff
@@ -179,6 +179,7 @@ List<Provider<RegularFile>> plugins = [
         systemProperty 'tests.rest.bwcsuite', 'old_cluster'
         systemProperty 'tests.rest.bwcsuite_round', 'old'
         systemProperty 'tests.plugin_bwc_version', bwcVersion
+        systemProperty 'tests.security.manager', 'false'
```

Comment:
I think this system property has no effect anymore

Reply:
Removed this in latest commit